gateway/changelog/unreleased.md

```diff
@@ -0,0 +1,30 @@
+# Unreleased
```

Comment:
I'm allowing myself to introduce this file, so we can write bits of release notes when we still have the context in mind, as opposed to rushing it at release time.